spring-boot/admin/pom.xml

```diff
@@ -252,9 +252,13 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.webjars.npm</groupId>
```

Comment:
We should probably use npm to manage JS dependencies (rather than webjars), to be further discussed.

In general, we should better update our existing font-awesome version, already included in the project, rather than adding a new version in parallel.